crates/extensions/c8y_mapper_ext/src/actor.rs

```diff
@@ -111,20 +123,28 @@ impl Actor for C8yMapperActor {
             .send(SyncStart::new(SYNC_WINDOW, ()))
             .await?;
 
-        while let Some(event) = self.messages.recv().await {
-            match event {
-                C8yMapperInput::MqttMessage(message) => {
-                    self.process_mqtt_message(message).await?;
-                }
-                C8yMapperInput::FsWatchEvent(event) => {
-                    self.process_file_watch_event(event).await?;
-                }
-                C8yMapperInput::SyncComplete(_) => {
-                    self.process_sync_timeout().await?;
-                }
-                C8yMapperInput::PublishMessage(message) => {
+        loop {
+            select! {
+                // in some tests may not be connected to the sender, so we need to ignore `None`
```

Comment:
```suggestion
                // in some tests may not be connected to the sender, so we need to ignore `None`
```
What may not be connected to the sender?

Reply:
Corrected the comment to mention the AvailabilityActor